app/uk/gov/hmrc/teamsandrepositories/services/RebuildService.scala

```diff
@@ -49,10 +53,25 @@ case class RebuildService @Inject()(
         Unit
       } else {
         val oldestJob = jobs.head
-        jenkinsService.triggerBuildJob(oldestJob.service, oldestJob.jenkinsURL, oldestJob.lastBuildTime)
+        for {
+          build <- jenkinsService.triggerBuildJob(oldestJob.service, oldestJob.jenkinsURL, oldestJob.lastBuildTime)
+          _ <- sendBuildFailureAlert(build, oldestJob.service) if build.result.nonEmpty && build.result.get.equalsIgnoreCase("FAILED")
```

Comment:
should we move the model states to the model?

Reply:
Done